packages/gateway/package.json

```diff
@@ -25,6 +25,7 @@
     "@graphql-tools/graphql-file-loader": "^6.0.9",
```

Comment:
Sorry, these Gateway packages shouldn't be in this PR. My bad. I'll remove these asap